org-code-javabuilder/playground/src/main/java/org/code/playground/Item.java

```diff
@@ -1,31 +1,51 @@
 package org.code.playground;
 
-interface Item {
+public abstract class Item {
```

Comment:
what's the reasoning for making this an abstract class?

Reply:
(instead of a regular class that is)

Reply:
Could be totally misunderstanding things, but I thought abstract classes were ones that you do not want to be instantiated directly, but rather subclassed (and then instantiated via those subclasses). Isn't that what we want in this case?

Reply:
Also updated to not public b/c I don't think we want students to be able to subclass it directly?

Reply:
yep you're right on abstract, I was thinking since it didn't have abstract methods it shouldn't be abstract but that's not the case. I think we do want students to be able to reference it like `Item x = new TextItem()`. Will that still work with not making it public?

Reply:
mmm not sure I understand that syntax, what does it mean to create an object and mark its type as it's parent? I believe I just tested `TextItem x = new TextItem()`?

Reply:
It's a pretty common java pattern, it just means that it is some subclass of the parent (or the parent itself it it's not an abstract class).

Reply:
Got it, and if you mark the object as type `Item` you wouldn't have access to methods defined on the subclass, yeah? It looks like it works if I mark the class as `public abstract` rather than just `abstract`

Reply:
yep!

Reply:
Alrighty thanks! Made Item public then :)